pytorch_forecasting/models/nhits/_nhits_pkg.py

```diff
@@ -84,19 +82,21 @@ def _get_test_dataloaders_from(cls, params):
             dataloaders_fixed_window_without_covariates,
             make_dataloaders,
         )
-        from pytorch_forecasting.tests._loss_mapping import DISTR_LOSSES_NUMERIC
-
-        distr_losses = tuple(
-            type(l)
-            for l in DISTR_LOSSES_NUMERIC
-            if not isinstance(l, MultivariateNormalDistributionLoss)
-            # use dataloaders without covariates as default settings of nhits
-            # (hidden_size = 512) is not compatible with
-            # MultivariateNormalDistributionLoss causing Cholesky
-            # decomposition to fail during loss computation.
-        )
 
-        if isinstance(loss, distr_losses):
+        # Use fixed window dataloaders for MultivariateNormalDistributionLoss
+        if isinstance(loss, MultivariateNormalDistributionLoss):
```

Comment:
can this explicit checking now be replaced by checking only tags of the metrics?

If not, can we think about what tag could replace this?